ci/cron/src/Main.hs

```diff
@@ -207,25 +216,24 @@ build_docs_folder path versions latest = do
             -- Starting after 0.13.54, we have changed the way in which we
             -- trigger releases. Rather than releasing the current commit by
             -- changing the VERSION file, we now mark an existing commit as the
-            -- source code for a release by changing the LATEST file. However,
-            -- release notes still need to be taken from the release commit
-            -- (i.e. the one that changes the LATEST file, not the one being
-            -- pointed to).
+            -- source code for a release by changing the LATEST file. This
+            -- raises the question of the release notes: as we tag a commit
+            -- from the past, and keep the changelog outside of the worktree
+            -- (in commit messages), that means that commit cannot contain its
+            -- own release notes. We have decided to resolve that conundrum by
```

Comment:
“from the most recent release” isn’t quite right as far as I can tell. You take it from the state of `master` and then build the `.html` file of the release for which you do not have docs (in theory there could be more than one and then you’ll end up with a somewhat random one but there shouldn’t be a meaningful difference since the state of `master` is the same). Any reason why we are not using the release commit?

Reply:
Under the (unverified, but so far true) assumption that we do not change the release notes file outside of a release cycle (and that if we do, then there is no release so this script will not run until there actually is a release), the latest commit that changes the release notes on master _is_ the release commit of the latest release. This could only be a different commit if we committed changes to the release notes file in-between merging a release commit to master and the cron job running. This could be actually useful for last-minute typo corrections I suppose.

This could be made slightly safer by pinning it to the LATEST file instead of the release notes one.

Reply:
It’s not true if you look at https://github.com/digital-asset/daml/commits/master/docs/source/support/release-notes.rst. I don’t think we’ve changed it sufficiently quickly that it would matter but it can change.

Reply:
Changed to check LATEST, which should be closer to the last release. I don't really want to duplicate the entire logic of determining whether that commit was a release commit, especially given that this only runs if a version was actually added to GitHub.